Cargo.toml

```diff
@@ -12,6 +12,7 @@ members = [
     "vm/message",
     "vm/runtime",
     "vm/state_tree",
+    "vm/state_manager",
```

Comment:
doesn't make much sense to me for the state manager to be inside of the vm folder, since the state manager is external to the vm? Thoughts from anyone else?

Reply:
Yeah that works for me, I put it there because of the state managers interactions with the state tree, and actors but could definitely be moved into blockchain component.

Reply:
Obviously no functional change, but organizationally it makes more sense to me